src/en/animeowl/src/eu/kanade/tachiyomi/animeextension/en/animeowl/AnimeOwl.kt

```diff
@@ -46,7 +46,7 @@ class AnimeOwl : ConfigurableAnimeSource, ParsedAnimeHttpSource() {
 
     override val name = "AnimeOwl"
 
-    override val baseUrl by lazy { preferences.getString("preferred_domain", "https://animeowl.net")!! }
+    override val baseUrl by lazy { preferences.getString("preferred_domain", "https://anime-owl.net")!! }
```

Comment:
Look at lines 340-342. The preference still sets the domain to animeowl.net. Why does the preference even exist if there appears to be only one domain?

Reply:
Don't click `Blame` please